dockstore-webservice/src/main/java/io/dockstore/webservice/core/Tag.java

```diff
@@ -98,15 +98,19 @@ public String getWorkingDirectory() {
     }
 
     public void updateByUser(final Tag tag) {
+        if (!this.isFrozen()) {
+            // this.setName(tag.getName());
+            imageId = tag.imageId;
```

Comment:
This applies to all the subsequent lines in this block, but what if tag.imageId is different than imageId?

Instead of ignoring like you do now, maybe this should return a 400 status code, otherwise users might think what they specified was applied.

OTOH, this is sort of minor, and the UI will probably be the only client of this to begin with, so if you agree it's the right way to handle this, it could wait for a new PR in another release.

Reply:
That sounds reasonable, I've created https://github.com/dockstore/dockstore/issues/2615 to track this.